### PR TITLE
IAM: introduce extended introspection endpoint

### PR DESCRIPTION
**Pull request by reinkrul**
Fixes https://github.com/nuts-foundation/nuts-node/issues/3179
Fixes https://github.com/nuts-foundation/nuts-node/issues/3178: specifies as VCs and VPs being in either string or object format.

### CODE REVIEW

File: docs/_static/auth/v2.yaml [hunk 1]
```diff
@@ -545,19 +573,22 @@ components:
...
-            $ref: '#/components/schemas/PresentationSubmission'
-          description: |
-            Mapping of Presentation Definition IDs that were fulfilled to Presentation Submissions.
-        vps:
-          type: array
-          items:
-            $ref: '#/components/schemas/VerifiablePresentation'
-          description: The Verifiable Presentations that were used to request the access token using the same encoding as used in the access token request.
+    ExtendedTokenIntrospectionResponse:
+      allOf:
```
Comment (woutslakhorst) [inline docs/_static/auth/v2.yaml:577]: How's your Java client holding up?

File: docs/_static/auth/v2.yaml [hunk 2]
```diff
@@ -144,6 +144,34 @@ paths:
...
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: "#/components/schemas/TokenIntrospectionRequest"
+      responses:
+        '200':
+          description: |
+            An Introspection response as described in RFC7662 section 2.2. The Irma, Dummy and Employee identity means all return 'username', 'initials',  'prefix', 'family_name' and 'assurance_level'.
```
Comment (woutslakhorst) [inline docs/_static/auth/v2.yaml:165]: Good time to update this description since the v2 api doesn't support these.

**Review by woutslakhorst — CHANGES_REQUESTED** [review]
update desc

File: docs/_static/auth/v2.yaml (hunk 1 above)
Comment (reinkrul) [reply docs/_static/auth/v2.yaml:577]: I'm getting it ready for v6, I'll make sure to test it.

Comment (reinkrul) [reply docs/_static/auth/v2.yaml:577]: Tested, had to define VerifiablePresentation as `string` (JWT) or `object` (JSON-LD).

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**